sdks/go/pkg/beam/core/runtime/exec/combine_test.go

```diff
@@ -148,7 +96,14 @@ func getCombineEdge(t *testing.T, cfn interface{}, ac *coder.Coder) *graph.Multi
 	}
 
 	g := graph.New()
-	inT := typex.NewCoGBK(typex.New(reflectx.Int), typex.New(reflectx.Int))
+	var vtype reflect.Type
+	if fn.AddInputFn() != nil {
+		// This makes the assumption that the AddInput function is unkeyed.
```

Comment:
As in it's unimplemented since it's a helper for a test, and none of the test cases exercise that at present. As implemented the CreateAccumulator and the AddInput functions are permitted to take a key as input.